router/image/templates/nginx.conf

```diff
@@ -15,43 +15,47 @@ http {
     sendfile on;
     tcp_nopush on;
     tcp_nodelay on;
-
     # The Timeout value must be greater than the front facing load balancers timeout value.
     # Default is the deis recommended timeout value for ELB - 1200 seconds + 100s extra.
-    {{ $defaultTimeout := or (.deis_router_defaultTimeout) "1300" }}
```

Comment:
> we can make this easier on ourselves by introducing a function like kelseyhightower/memkv#4 into our confd fork

+1. This really needs to be addressed soon upstream, but I have high confidence that useful changes like that will be merged. We really want to make the templates more readable, to lower the bar for contributors. Plus we are already using our own fork, so we have crossed that bridge. :bridge_at_night: